WalletWasabi.Fluent/Views/NavBar/NavBar.axaml

```diff
@@ -112,7 +115,7 @@
       <Setter Property="Data" Value="{Binding IconNameFocused, Converter={x:Static conv:NavBarIconConverter.Instance}}" />
     </Style>
   </UserControl.Styles>
-  <DockPanel Margin="0 -5 0 5">
+  <DockPanel Margin="0 0 0 5">
```

Comment:
Just curious why?

Reply:
Applying a negative margin caused the upper button to be clipped:

![image](https://user-images.githubusercontent.com/3109851/172592556-61a3302a-5a1d-4a2b-b5b8-f7526644bc27.png)

Overall, it the button looked 5 pixels smaller than the bottom one (down arrow).

I'm not sure why the margin was -5 in the first place. My guess is that this is added to align the navbar top side with the content top side. In my opinion, this is not needed and can potentially cause issues like this.

I'd prefer to keep it to 0. **BUT** we can restore it to -5 because the buttons are not visible for now, and we will not have the problem. However, if we want to enable arrow scrollers in the future, we will have to remember to apply the non-negative margin here.